src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -331,47 +331,46 @@
                                   [idx item])
                                 @(re-frame/subscribe [:chat/input-with-mentions]))]
          ^{:key (str idx "_" type "_" text)}
-         [rn/text (when (= type :mention) {:style {:color "#0DA4C9"}})
-          text])
+         [rn/text (when (= type :mention) {:style {:color (:text-04 @colors/theme)}}) text])
        (get @input-texts chat-id))]))
 
 (defn mention-item
   [[public-key {:keys [alias name nickname] :as user}] _ _ text-input-ref]
   (let [ens-name? (not= alias name)]
-    [list-item/list-item
-     (cond-> {:icon              [photos/member-photo public-key]
-              :size              :small
-              :text-size         :small
-              :title
-              [text/text
-               {:weight          :medium
-                :ellipsize-mode  :tail
-                :number-of-lines 1
-                :size            :small}
-               (if nickname
-                 nickname
-                 name)
-               (when nickname
-                 [text/text
-                  {:weight         :regular
-                   :color          :secondary
-                   :ellipsize-mode :tail
-                   :size           :small}
-                  " "
-                  (when ens-name?
-                    "@")
-                  name])]
-              :title-text-weight :medium
-              :on-press
-              (fn []
-                (re-frame/dispatch [:chat.ui/select-mention text-input-ref user]))}
-
-       ens-name?
-       (assoc :subtitle alias))]))
+    [rn/touchable-opacity
+     {:on-press #(re-frame/dispatch [:chat.ui/select-mention text-input-ref user])}
+
+     [list-item/list-item
+      (cond-> {:icon              [photos/member-photo public-key]
+               :size              :small
+               :text-size         :small
+               :title
+               [text/text
+                {:weight          :medium
+                 :ellipsize-mode  :tail
+                 :number-of-lines 1
+                 :size            :small}
+                (if nickname
+                  nickname
+                  name)
+                (when nickname
+                  [text/text
+                   {:weight         :regular
+                    :color          :secondary
+                    :ellipsize-mode :tail
+                    :size           :small}
+                   " "
+                   (when ens-name?
+                     "@")
+                   name])]
+               :title-text-weight :medium}
+
+        ens-name?
+        (assoc :subtitle alias))]]))
 
 (def chat-toolbar-height (reagent/atom nil))
 
-(defn autocomplete-mentions [text-input-ref bottom]
+(defn autocomplete-mentions-old [text-input-ref bottom]
```

Comment:
I guess this is where the migration strategy we discussed on Tuesday needs to start taking effect, 

If I recall correctly we want to separate old and new via the namespace, as such these additions should be in a different file.
However, I don't think we completely agreed on the how we divide these namespaces out. 
Can we make a decision so that it can be followed in this p.r (if needed)
@Parveshdhull  @flexsurfer  @ibrkhalil @ilmotta  @jo-mut